packages/ui/src/1_atoms/Paragraph/Paragraph.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+
+import React from 'react';
+
+import { Paragraph } from './Paragraph';
+import { ParagraphSize, ParagraphStyle } from './Paragraph.types';
+
+describe('Paragraph', () => {
+  it('renders a paragraph', () => {
+    render(<Paragraph children="Testing paragraph" />);
+    expect(screen.getByText('Testing paragraph')).toBeInTheDocument();
+  });
+
+  it('check paragraph instance', () => {
+    render(<Paragraph children="test" />);
+    const paragraph = screen.getByText('test');
+    expect(paragraph).toBeInstanceOf(HTMLParagraphElement);
+  });
+
+  it('should render a paragraph size with a className equal to the base', () => {
+    render(<Paragraph children="Paragraph size" size={ParagraphSize.base} />);
+    const paragraphClasses = screen
```

Comment:
```suggestion
    const classes = screen
```
Sorry for changing it again but I just realized that since we write tests for `Paragraph`, it's unnecessary to call it `paragraphClasses`.